src/DDTrace/OpenTelemetry/Span.php

```diff
@@ -136,7 +160,8 @@ public static function startSpan(
         array $attributes,
         array $links,
         int $totalRecordedLinks,
-        bool $isRemapped = true // Answers the question "Was the span created using the OTel API?"
+        bool $isRemapped = true, // Answers the question "Was the span created using the OTel API?"
+        array $events
```

Comment:
Doesn't really make sense to put the optional arg before the required one:
```suggestion
        array $events,
        bool $isRemapped = true // Answers the question "Was the span created using the OTel API?"
```